### PR TITLE
FeedCofense: 'test' button always returned a positive answer

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24794

## Description
Fixed a bug where the 'test' button always returned a positive 
answer.

## Screenshots
before:
image
after:
image

## Minimum version of Demisto
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] No

### CODE REVIEW

File: Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ script:
     description: Gets indicators from the feed.
     execution: false
     name: cofense-get-indicators
-  dockerimage: demisto/python3:3.8.2.6981
+  dockerimage: demisto/python3:3.8.2.8347
```
Comment (Itay4) [inline Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.yml:120]: this docker image does not exist

**Comment by Itay4** [comment]
need to check credentials is not none in the main func

**Comment by kirbles19** [comment]
@jochman Done.

File: Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py [hunk 2]
```diff
@@ -304,6 +305,9 @@ def main():
     url = "https://www.threathq.com"
     credentials = params.get("credentials", {})
     auth = (credentials.get("identifier"), credentials.get("password"))
+    if not auth[0] or not auth[1]:
```
Comment (Itay4) [inline Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py:308]: this is not what i meant
credentials are not mandatory
the code should be something like that:
```python
credentials = params.get("credentials", {}) or {} 
```

**Pull request merged**